### PR TITLE
reduced comments and changed variable names for method grow

### PR DESCRIPTION
**Pull request by jaykay99992**
The comment for method grow() was too lengthy so I reduced it. And I changed the variable names v and h to VE and HE to give a clearer abbreviation

### CODE REVIEW

File: jhotdraw-utils/src/main/java/org/jhotdraw/geom/Geom.java [hunk 1]
```diff
@@ -620,30 +620,26 @@ public static double distanceFromLine(int xa, int ya, int xb, int yb, int xc, in
...
-   * @param v the vertical expansion
+   * @param HE the horizontal expansion
+   * @param VE the vertical expansion
    */
-  public static void grow(Rectangle2D.Double r, double h, double v) {
-    r.x -= h;
-    r.y -= v;
-    r.width += h * 2d;
-    r.height += v * 2d;
+  public static void grow(Rectangle2D.Double r, double HE, double VE) {
```
Comment (JFLeblay) [inline jhotdraw-utils/src/main/java/org/jhotdraw/geom/Geom.java]: In Java, the convention is to use capital-only variable names for constant (static final).
In your case, HE and VE should be horizontalExpansion and verticalExpansion.
This makes your method signature clearer for the programmer who would use it in his/her own code.

**Review by JFLeblay — CHANGES_REQUESTED** [review]
Please change the code according to my comments so that I can approve the Pull Request.

**Comment by jaykay99992** [comment]
@JFLeblay I just updated the variable names as you mentioned to represent what they do clearly. Is it fine like this ? Do i merge it like this ?

File: jhotdraw-utils/src/main/java/org/jhotdraw/geom/Geom.java [hunk 2]
```diff
@@ -620,30 +620,26 @@ public static double distanceFromLine(int xa, int ya, int xb, int yb, int xc, in
...
-   * larger on both the left and right side, and <code>v</code> units larger at both the top and
+   * <p>This method modifies the Rectangle2D.Double so that it is horizontal expansion units larger
+   * on both the left and right side, and vertical expansion units larger at both the top and
    * bottom.
    *
-   * <p>The new <code>Rectangle2D.Double</code> has (<code>x&nbsp;-&nbsp;h</code>, <code>
-   * y&nbsp;-&nbsp;v</code>) as its top-left corner, a width of <code>width</code>&nbsp;<code>+
-   * </code>&nbsp;<code>2h</code>, and a height of <code>height</code>&nbsp;<code>+</code>&nbsp;
-   * <code>2v</code>.
+   * <p>The new Rectangle2D.Double grows is if HE and VE are positive,
```
Comment (JFLeblay) [inline jhotdraw-utils/src/main/java/org/jhotdraw/geom/Geom.java]: Please make sure you use meaning names and not abbreviations for HE and VE

File: jhotdraw-utils/src/main/java/org/jhotdraw/geom/Geom.java [hunk 3]
```diff
@@ -620,30 +620,26 @@ public static double distanceFromLine(int xa, int ya, int xb, int yb, int xc, in
...
    *
-   * <p>If negative values are supplied for <code>h</code> and <code>v</code>, the size of the
-   * <code>Rectangle2D.Double</code> decreases accordingly. The <code>grow</code> method does not
-   * check whether the resulting values of <code>width</code> and <code>height</code> are
-   * non-negative.
+   * <p>If negative values are supplied for HE and VE, the size of the Rectangle2D decreases
```
Comment (JFLeblay) [inline jhotdraw-utils/src/main/java/org/jhotdraw/geom/Geom.java]: Please make sure you use meaning names and not abbreviations for HE and VE

**Review by JFLeblay — CHANGES_REQUESTED** [review]
When you are refactoring, please make sure the documentation reflects your changes. In your case, you have not replace HE and VE with meaningful names in the documentation.

**Comment by jaykay99992** [comment]
@JFLeblay yeaa my bad, didn't see those. I just updated them too. Should I merge it like this now ?

**Review by JFLeblay — APPROVED** [review]
No problem.
You're good now. Please merge.

**Comment by jaykay9999** [comment]
good job, code merged

**Pull request merged**